kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -425,8 +427,8 @@ pub struct LookupConfiguration<F: FftField> {
     /// This is used to pad the lookups to `max_lookups_per_row` when fewer lookups are used in a
     /// particular row, so that we can treat each row uniformly as having the same number of
     /// lookups.
-    #[serde_as(as = "JointLookup<o1_utils::serialization::SerdeAs>")]
-    pub dummy_lookup: JointLookup<F>,
+    #[serde_as(as = "JointLookupValue<o1_utils::serialization::SerdeAs>")]
+    pub dummy_lookup: JointLookupValue<F>,
```

Comment:
the name still confused me today. If it's joint, then why is it still a vec :)

Reply:
I suppose we should standardise on e.g. 'joint' meaning that it represents multiple values, and 'joined' meaning those multiple values have been combined using the joint combiner.